src/components/MoviesList/MovieBox/MovieBox.jsx

```diff
@@ -1,24 +1,118 @@
 import React from "react";
 import PropTypes from "prop-types";
+import "./movie-box.scss"
+import MovieDropdownList from "../MovieDropdownList/MovieDropdownList";
+import EditMovieForm from "../../Modal/ModalContent/EditMovieForm/EditMovieForm";
+import Modal from "../../Modal/Modal";
+import DeleteMovie from "../../Modal/ModalContent/DeleteMovie/DeleteMovie";
 
-const MoviesBox = (props) => {
-    const {poster_path, title, genres, release_date} = props.movie;
+export default class MoviesBox extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            showButtonDropdown: false,
+            showDropdown: false,
+            isShowModalEdit: false,
+            isShowModalDelete: false
+        }
+    }
 
-    return(
-        <div className="movie-box">
-            <img src={poster_path} alt=""/>
-            <div className="movie-description">
-                <div className="movie-heading">
-                    <h2>{title}</h2>
-                    <span className="movie-title">{genres.join(', ')}</span>
+    showButtonDropdown = () => {
+        this.setState({
+            showButtonDropdown: true
+        })
+    };
+
+    hideButtonDropdown = () => {
+        this.setState({
+            showButtonDropdown: false,
+            showDropdown: false
+        })
+    };
+
+    showDropdown = () => {
+        this.setState({
+            showDropdown: true
+        })
+    };
+
+    hideDropdown = () => {
+        this.setState({
+            showDropdown: false
+        })
+    };
+
+    handleShowModalEdit = () => {
+      this.setState({
+          isShowModalEdit: true
+      })
+    };
+
+    handleShowModalDelete = () => {
+        this.setState({
+            isShowModalDelete: true
+        })
+    };
+
+    handleCloseModalDelete = () => {
+        this.setState({
+            isShowModalDelete: false
+        })
+    };
+
+    handleCloseModalEdit = () => {
+        this.setState({
+            isShowModalEdit: false
+        })
+    };
+
+    render() {
+        const {poster_path, title, genres, release_date} = this.props.movie;
+
+        return (
+            <>
+                <div className="movie-box" onMouseEnter={this.showButtonDropdown} onMouseLeave={this.hideButtonDropdown}>
```

Comment:
quick question: can't it be done with css only? 🤔

Reply:
yes, thanks, I moved hover to css)